modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+= Installing cert-manager
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+You can integrate `cert-manager` with {SMProduct} by deploying `istio-csr` and then creating an `Istio` resource that uses `istio-csr` to process workload and `controlplane` certificate signing requests. This example creates a self signed `Issuer`, but any other `Issuer` can be used instead.
+
+[IMPORTANT]
+====
+The cert-manager tool must be installed before you create and install your `Istio` resource.
+====
+
+.Procedure
+
+. Create the `istio-system` namespace:
++
+[source, terminal]
+----
+$ oc create namespace istio-system
+----
+
+. Create the root issuer:
++
+.. Create the `Issuer` object as in the following example:
++
+.Example `issuer.yaml`
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned
+  namespace: istio-system
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+    name: istio-ca
+    namespace: istio-system
+spec:
+  isCA: true
+  duration: 87600h # 10 years
+  secretName: istio-ca
+  commonName: istio-ca
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  subject:
+    organizations:
+      - cluster.local
+      - cert-manager
+  issuerRef:
+    name: selfsigned
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: istio-ca
+  namespace: istio-system
+spec:
+  ca:
+    secretName: istio-ca
+---
+----
++
+.. Create the objects using the following command:
++
+[source, terminal]
++
+----
+$ oc apply -f issuer.yaml
+----
++
+.. Wait for the `istio-ca` certificate to become ready:
```

Comment:
```suggestion
.. Wait for the `istio-ca` certificate to become ready by running the following command:
```
Is the step here to wait, or should another verb be used here? It seems odd to say "Wait...by running the following command:"

Reply:
The command is: +$ oc wait --for=condition=Ready certificates/istio-ca -n istio-system

what would you suggest be used instead of `wait` , then?

Reply:
If I knew more about what happens after you run the command, I'd be able to provide a better suggestion. It just caught me as odd that someone would enter a command in order to wait.

Reply:
They have to wait for the `istio-ca` cert to be ready, and then they can go to the next step and copy the `istio-ca` certificate to the `cert-manager` namespace, and continue. They run the get command to copy the secret, and then the create command to create the secret.

Reply:
Peanut gallery comment here:
Maybe the docs for the `kubectl wait` command would help you with improved phrasing? https://kubernetes.io/docs/reference/kubectl/generated/kubectl_wait/#synopsis

Reply:
The Peanut Gallery! :laughing: I haven't heard that phrase in a long, long time.

Reply:
Thanks @michaelryanpeter!